addons/web_editor/static/src/js/editor/odoo-editor/src/utils/sanitize.js

```diff
@@ -187,8 +187,9 @@ function sanitizeNode(node, root) {
         node = parent; // The node has been removed, update the reference.
     } else if (
         node.nodeName === 'P' && // Note: not sure we should limit to <p>.
-        node.parentElement.nodeName === 'LI' &&
-        !node.parentElement.classList.contains('nav-item')
+        ['LI', 'A'].includes(node.parentElement.nodeName) &&
+        !node.parentElement.classList.contains('nav-item') &&
+        !node.parentElement.matches('nav a, a.nav-link')
```

Comment:
To check.

Reply:
done !

Reply:
**UPD** : There was a test failing when I remove the nav-link check here, test name: `test_15_website_link_tools`

Reply:
**UPD:** I changed the the test which was failing because we use `p` under `a` in `s_mega_menu_cards_item` template.